test/unit/path_spec.lua

```diff
@@ -293,6 +293,60 @@ describe('path_shorten_fname_if_possible', function()
   end)
 end)
 
+describe('path.c path_guess_exepath', function()
+  local cwd = lfs.currentdir()
+
+  for _,name in ipairs({'./nvim', '.nvim', 'foo/nvim'}) do
+    itp('"'..name..'" returns name catenated with CWD', function()
+      local name = to_cstr(name)
```

Comment:
@justinmk Strange, see e.g. https://github.com/neovim/neovim/blob/17531ed0825c1bf5e24e78ae94020fb08ca06013/test/unit/eval/typval_spec.lua#L1636. Maybe it needs `const` to do automatic conversion?